### DISCUSSION TITLE
Add a wordpress filter on get_settings_for_display

### DISCUSSION THREAD

**Discussion by Tofandel** [elementor/elementor#22753] [Feature Request]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

Ìn wordpress shortcode attributes can normally be filtered and modified by themes or plugins, elementor does not provide this feature

This normally allows for exemple in adding placeholders for links or fonts that can be replaced or set by default in the filter

### Describe the solution you'd like

A filter named `elementor/{$element_type}/get_settings_for_display` in the `get_settings_for_display` method
https://github.com/elementor/elementor/blob/a61ffff6fef34764d6ead15b2180a64bcd61c23b/includes/base/controls-stack.php#L1210-L1216

Here is how it could be implemented
```php
if ( ! $this->parsed_active_settings ) { 
    $settings = $this->get_active_settings( $this->get_parsed_dynamic_settings(), $this->get_controls() ); 
    $element_type = $this->get_type();
    $this->parsed_active_settings = apply_filters( "elementor/{$element_type}/get_settings_for_display", $settings, $this );
} 
```

**Comment by imflexwala** [comment]
Please add this feature
it would be very help to handle programming cases where dynamic data is required
